.github/workflows/build.yml

```diff
@@ -265,13 +265,13 @@ jobs:
           key: ${{ matrix.os }}-${{ matrix.EVENT_MATRIX }}-v4
 
       - name: Prepare vcpkg
-        uses: lukka/run-vcpkg@v11
+        uses: lukka/run-vcpkg@v7
         id: runvcpkg
         with:
           vcpkgArguments: zlib:x64-windows openssl:x64-windows mbedtls:x64-windows
           vcpkgDirectory: ${{ runner.workspace }}/vcpkg/
           vcpkgTriplet: x64-windows
-          vcpkgGitCommitId: c8696863d371ab7f46e213d8f5ca923c4aef2a00  # 2023.12.12
+          vcpkgGitCommitId: af2287382b1991dbdcb7e5112d236f3323b9dd7a
```

Comment:
In my repository, I tested only moving from `lukka/run-vcpkg@v11` back to `lukka/run-vcpkg@7` and did not touch the `vcpkgGitCommitId`. And all related CI jobs passed -- https://github.com/hebasto/libevent/actions/runs/9036824192.

Reply:
But how `vcpkgGitCommitId` works then in this case?

Reply:
vcpkg at the old commit https://github.com/microsoft/vcpkg/commit/af2287382b1991dbdcb7e5112d236f3323b9dd7a uses the outdated `libwinpthread` version, which is unavailable for downloading now (https://github.com/microsoft/vcpkg/issues/38263).

The updated vcpkg (the current commit https://github.com/microsoft/vcpkg/commit/c8696863d371ab7f46e213d8f5ca923c4aef2a00 seems enough) uses the new `libwinpthread`.

Reply:
Thanks! I've applied your commit instead in this PR